ui/app/adapters/pki/issuer.js

```diff
@@ -31,6 +33,45 @@ export default class PkiIssuerAdapter extends ApplicationAdapter {
     }
   }
 
+  async getIssuerMetaData(store, type, query, res) {
```

Comment:
nit: I think metadata is one word 
```suggestion
  async getIssuerMetadata(store, type, query, res) {
```